src/components/App/Header/InboxContent.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useMemo, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+import { formatDistance } from 'date-fns';
+import { zhTW } from 'date-fns/locale';
+
+import popoverStyles from './Header.module.css';
+import styles from './InboxContent.module.css';
+import { messagesSelector } from 'selectors/inbox';
+import { readInbox } from 'actions/inbox';
+
+const InboxContent = ({ className, isOpen = true }) => {
+  const allMessages = useSelector(messagesSelector);
+
+  const [showsUnread, setShowsUnread] = useState(false);
```

Comment:
nit: `showUnread` or `onlyShowUnread`

Reply:
It's a boolean value so I want show"s" to imply that